src/zenml/models/v2/base/filter.py

```diff
@@ -598,6 +640,23 @@ def _resolve_operator(value: Any) -> Tuple[Any, GenericFilterOps]:
             ):
                 value = split_value[1]
                 operator = GenericFilterOps(split_value[0])
+
+            if operator == operator.ONEOF:
+                try:
+                    value = json.loads(value)
+                    if not isinstance(value, list):
+                        raise ValueError(
+                            "When you are using the 'oneof:' filtering "
+                            "make sure that the provided value is a json "
+                            "formatted list."
+                        )
```

Comment:
```suggestion
                        raise ValueError
```
No need to duplicate the text

Reply:
Fixed this as well.